src/mscorlib/src/System/Span.cs

```diff
@@ -625,5 +636,266 @@ internal static unsafe void CopyTo<T>(ref T destination, ref T source, int eleme
                 }
             }
         }
+
+        internal static unsafe void ClearWithoutReferences(ref byte b, nuint byteLength)
+        {
+            if (byteLength == 0)
+                return;
+
+            fixed (byte* bytePointer = &b)
```

Comment:
You should rewrite this using `ref byte`, without pinning. The pinning is unnecessary overhead - especially for small sizes.